omnigan/losses.py

```diff
@@ -303,6 +306,15 @@ def __call__(self, input, target, mask):
         return torch.mean(torch.abs(torch.mul((input - target), 1 - mask)))
 
 
+class CompareLoss(nn.Module):
```

Comment:
I think CompareLoss is not explicit enough, something like `GroundIntersectionLoss` or something, it's ok if it's not generic it's really special for the ground in our use case